Guide updates for Entra conditional access (#44095)
fleetdm/fleet: articles/conditional-access.md

# Conditional access

Fleet's conditional access feature lets IT and security teams enforce access controls on macOS and Windows hosts based on policy status. When a host fails a particular policy in Fleet, access to third-party apps can be blocked until the issue is resolved.

## How it works

1. IT enables the conditional access automation for the policies which determine access.
2. Fleet evaluates policies on each host.

> 3. Fleet communicates compliance status to the IdP (directly for Okta, via Intune for Entra).

Reviewer: ```suggestion
3. Fleet communicates compliance status to the identity provider (IdP).
```

It's technically straight to Entra for Windows, right? Either way, I think we can remove this bit.

4. The IdP enforces access decisions, blocking users who are failing the policies from logging into protected apps.
5. Users remediate issues on their hosts and refetch to verify. Once the host passes all required policies, access is restored.

## Setup

Fleet currently has built-in conditional access integrations with Okta (macOS) and Entra (macOS and Windows).

For detailed setup instructions and to preview end user experience, see the guide for your identity provider:

- [Okta setup guide](https://fleetdm.com/guides/okta-conditional-access-integration)
- [Entra setup guide](https://fleetdm.com/guides/entra-conditional-access-integration)

<meta name="articleTitle" value="Conditional access">
<meta name="authorFullName" value="Rachael Shaw">
<meta name="authorGitHubUsername" value="rachaelshaw">
<meta name="category" value="guides">

> <meta name="publishedOn" value="2026-04-22">

Author: ```suggestion
<meta name="publishedOn" value="2026-04-27">
```
